ui/src/app/styles/colors.ts

```diff
@@ -11,6 +11,7 @@ export default {
   success: '#8BC990',
   // Used for tables
   light: '#E9ECEF',
+  lightBackground: '#E6E4Ed',
```

Comment:
Is this in our official list of colors? If not, we should ask Lou to add it in. Or maybe `light` is the color Lou wanted?